docker-build-components-example.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+: '
+Make a copy of this file and rename to docker-build-components.sh
+as the renamed file is in .gitignore. This will avoid commiting the file
+with certificate information.
+'
+echo "Exporting environment variables"
+#Put your party key here (OpenTest):
+export MHS_SECRET_PARTY_KEY=""
```

Comment:
do all of these variables need to be exported or none of them?

Reply:
Should all be exported as they are used in the docker compose